.npmrc

```diff
@@ -1,4 +1,7 @@
 ; # START SKY UX OVERRIDES - DO NOT MODIFY
 ; The following overrides are required to support NPM 8 on the SKY UX ADO pipelines.
 legacy-peer-deps=true
+; # END SKY UX OVERRIDES; # START SKY UX OVERRIDES - DO NOT MODIFY
+; The following overrides are required to support NPM 8 on the SKY UX ADO pipelines.
+legacy-peer-deps=true
```

Comment:
Looks like this was duplicated. Probably a bug with SKY UX CLI.

Reply:
Removed the duplicate